app/src/comic/management/commands/seed_test_data.py

```
@@ -0,0 +1,50 @@
+import random
+
+from comic.models import Publisher, Series
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "產生 10,000 筆漫畫系列測試資料"
+
+    def handle(self, *args, **kwargs):
```

Ruff (0.14.10)

Unused method argument: `args`

(ARG002)